<launch>
  <node pkg="ccny_karto" type="scan_matcher_node" name="scan_matcher_node" output="screen">
    <param name="history_length" value="10"/>
    <param name="search_sizes" type="string" value="1.00 0.20"/>
    <param name="resolutions" type="string" value="0.05 0.01"/>
    <param name="history_skip" value="5"/>

    <param name="odom_frame" value="laser"/>
    <param name="base_frame" value="laser"/>
  </node>
</launch>
